src/App.jsx

```diff
@@ -122,6 +122,18 @@ class App extends Component {
     checkSyncLink()
 
     this.props.setLoading(false)
+
+    // Effectively delete databases
+    const dbsToDelete = JSON.parse(window.localStorage.getItem('dbsToDelete')) || []
+    console.log('dbsToDelete', dbsToDelete)
```

Comment:
Maybe adding debug(log) is better (debug is exported in utils)

Reply:
I forgot to delete this line, thanks